### PR TITLE
Login Page Font Mismatch Fix

### PR DESCRIPTION
**Pull request by efunnuga-bamidele**
Applied Lato Font to the Login Page and all other elements on the page, also made use of the !Important  CSS tag to override any other font style on the page.
image

### CODE REVIEW

File: packages/main/src/pages/login/Login.module.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
+#authPageWrapper * {
+  font-family: "Lato", sans-serif !important;
+}
+
 #authPageWrapper {
-  font-family: "Lato", sans-serif;
+  /* font-family: "Lato", sans-serif; */
```
Comment (JulianaSau) [inline packages/main/src/pages/login/Login.module.css:6]: If the code is not being used, you should do away with it entirely instead fo commenting it out

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/pages/login/Login.module.css (hunk 1 above)
Comment (JulianaSau) [reply packages/main/src/pages/login/Login.module.css:6]: Also, ensure you add screenshots of the component before and after your changes

**Comment by efunnuga-bamidele** [comment]
Code Before Changes
image

Code After Changes
image

**Comment by JulianaSau** [comment]
I mean screenshots of the page itself @efunnuga-bamidele. The code is okay.

**Comment by efunnuga-bamidele** [comment]
Before Code Change
image


After Code Change
image

**Review by JulianaSau — APPROVED** [review]
Looks good

**Comment by billmal071** [comment]
I don't see why you need to use !important, can you look at the style affecting it and removing so that there won't be a need to use important.

**Comment by efunnuga-bamidele** [comment]
@billmal071, changes implemented
Before Changes
before

After Changes
after

**Comment by JulianaSau** [comment]
@efunnuga-bamidele pull changes from the dev branch and push them to your remote repository

**Pull request merged**